concrete-core/src/backends/core/private/crypto/ggsw/ciphertext.rs

```diff
@@ -432,4 +462,220 @@ impl<Cont> GgswCiphertext<Cont> {
                 )
             })
     }
+
+    pub fn fill_with_trivial_encryption<Scalar>(&mut self, plaintext: &Plaintext<Scalar>)
```

Comment:
Here it would be good to have an overview of the algorithm (I don't know it so I can't really tell if what's below is correct).

Reply:
Will do!

Reply:
Actually I'm not sure this is the place to do this, this algorithm basically do the same thing than encrypt_constant_ggsw but with a trivial mask and without adding gaussian noise. If an overview is needed, I think a better place would be in private/crypto/secret/glwe.rs.

Reply:
Then I would go for that and put here a link to encrypt_constant_ggsw maybe?

Reply:
Will do.